src/main/resources/messages_de.properties

```diff
@@ -84,4 +83,22 @@ administration.statistics.month=Nach Monat
 administration.statistics.recent=Kürzlich
 administration.statistics.all_time=Alle
 bookings.start.value=Start: {0}
-bookings.end.value=Ende: {0}
\ No newline at end of file
+bookings.end.value=Ende: {0}
+mail.booking_deleted.heading= Deine Buchung wurde gelöscht.
```

Comment:
Why have you added whitespace before the value?
In any case, I'd recommend using the "Resource Bundle Editor" plugin for IDEA to edit these